packages/core/src/core.js

```diff
@@ -12,10 +12,24 @@ export class TSElement extends LitElement {
 	static get styles() {
 		return [unsafeCSS(commonCSS)];
 	}
-
+	// TODO: Remove this when changed rtl to dir in ohter components
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1135)